### ISSUE TITLE
请问genius-large-k2t模型不开源吗？我本地跑报错

### ISSUE THREAD

**Issue by Qi-Ming** [beyondguo/genius#17]

**Comment by Qi-Ming** [comment]
from transformers import pipeline
import torch
print(torch.__version__)
print(torch.cuda.is_available())

# 1. load the model with the huggingface `pipeline`
genius = pipeline("keywords-to-text", model='genius-large-k2t', device=0)
# 2. provide a sketch (joint by <mask> tokens)
sketch = "<mask> phtoto <mask> girl <mask> beautiful <mask>"
# 3. here we go!
generated_text = genius(sketch, num_beams=3, do_sample=True, max_length=200)[0]['generated_text']
print(generated_text)

**Comment by Qi-Ming** [comment]
genius-large-k2t is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'
If this is a private repository, make sure to pass a token having permission to this repo with `use_auth_token` or log in with `huggingface-cli login` and pass `use_auth_token=True`.
